app/models/node.rb

```diff
@@ -359,20 +359,6 @@ def scraped_image
     match&.split('src="')&.last&.split('"')&.first
   end
 
-  # was unable to set up this relationship properly with ActiveRecord associations
-  def drupal_content_field_image_gallery
-    DrupalContentFieldImageGallery.where(nid: nid)
-                                  .order('field_image_gallery_fid')
-  end
-
-  def gallery
```

Comment:
Yes, here, we are actually removing the method we need to use in the migration. We can use the definition itself to resolve this!